### DISCUSSION TITLE
Welcome to garbo Discussions!

### DISCUSSION THREAD

**Discussion by gausie** [loathers/garbage-collector#1619] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the garbo user community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by PharmerGord** [comment]
Thank you for opening up discussions I hope that this can be a great place to discuss ideas!

**Comment by asafaq** [comment]
Can I use garbo for neverending party meat farming? -> but I'm actually looking to finish 9 hard quests and get Drinking to Drink skill book automatically (since I haven't found a script just for NEP).

**Reply by gausie** [reply]
Perhaps the defining feature of this piece of software called garbage-collector is that it is used for collecting garbage in Dinseyland

**Reply by asafaq** [reply]
Oh that's Ok, I can do that too.
